scripts/utils/create_package_template.py

```diff
@@ -249,7 +250,8 @@ def package_version(dependency_version):
 Import-Module vm.common -Force -DisableNameChecking
 
 $pluginName = '{tool_name}'
-VM-Uninstall-IDA-Plugin -pluginName $pluginName
+$pluginFiles = @({files})
+VM-Uninstall-IDA-Plugin -pluginName $pluginName -pluginFiles $pluginFiles
```

Comment:
I do not like that we know always use the `pluginFiles` argument even if it is not needed. I think this complicates the code for the default case making it also confusing. What about adding an `if` to check if `files` is empty in `create_ida_plugin_template` and having two different templates (the previous one and the purposed one)?